### PR TITLE
📝 Update stand-up documentation

### PR DESCRIPTION
**Pull request by StatsRhian**
Following the discussion on Thursday with Chris & YiWen

### CODE REVIEW

File: style/team.qmd [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-If you are going to be absent on a Monday morning, you might want to share your three-things on Slack in advance.
+There is also a Slack bot every Wednesday & Friday prompting a check-in. You don't have to respond to this, but it's a nice way to share any recent wins or blockers.
+
+## Weekly stand up
+
+Every Monday from 10:00 - 10:50 we have a team meeting.
+
+We start with a stand-up from two team members. These are randomly selected by a bot on Slack in advance of the meeting.
+
+If it's your week to talk, you have 5 minutes to chat about your work. You could expand on your Slack stand-up, perhaps sharing your wins or learnings in more detail. You might also choose to share a project you are working on in, showing code or a shiny application. Chat about your weekend is also totally fine!
```
Comment (ChrisBeeley) [inline style/team.qmd:19]:
```suggestion
If it's your week to talk, you have 5 minutes to chat anything that matters to you. You could expand on your Slack stand-up, perhaps sharing your wins or learnings in more detail. You might also choose to share a project you are working on in, showing code or a shiny application. Chat about your weekend or an update on how you're doing or relating any domestic emergencies is also totally fine!
```

**Review by ChrisBeeley — APPROVED** [review]
Added a suggestion to make it clearer that people can just chat about themselves if they want, I think that's a really important part of it

File: style/team.qmd (hunk 1 above)
Comment (StatsRhian) [reply style/team.qmd:19]: I think "personal matters" rather than "domestic emergencies"?

Comment (StatsRhian) [inline style/team.qmd:19]:
```suggestion
If it's your week to talk, you have 5 minutes to chat about anything that matters to you. You could expand on your Slack stand-up, perhaps sharing your wins or learnings in more detail. You might also choose to share a project you are working on in, showing code or a shiny application. Chat about your weekend or an update on how you're doing is also totally fine!
```

**Pull request merged**